package.json

```diff
@@ -7,6 +7,7 @@
     "start": "node build/index.js",
     "dev": "cross-env NODE_ENV=development nodemon --watch './**/*.ts' --exec 'ts-node' index.ts",
     "test": "cross-env NODE_ENV=test jest --watch",
+    "posttest": "cross-env NODE_ENV=test ts-node test-cleanup.ts",
```

Comment:
This script will run after `test` if it is called `posttest` to clear all folders